src/tribler-core/tribler_core/upgrade/db8_to_db10.py

```diff
@@ -0,0 +1,116 @@
+import contextlib
+import datetime
+import logging
+import sqlite3
+from asyncio import sleep
+
+TABLE_NAMES = (
+    "ChannelNode", "TorrentState", "TorrentState_TrackerState", "ChannelPeer", "ChannelVote", "TrackerState", "Vsids")
```

Comment:
Should the `MiscData` table also be on this list?

Reply:
No. MiscData is currently only used for upgrader marks and database version.